src/test/java/com/wikia/webdriver/testcases/hubstests/HubsTests.java

```diff
@@ -15,12 +10,11 @@
 import com.wikia.webdriver.pageobjectsfactory.pageobject.WikiBasePageObject.HubName;
 import com.wikia.webdriver.pageobjectsfactory.pageobject.special.SpecialManageWikiaHome;
 
-/**
- * @author Karol 'kkarolk' Kujawiak
- * @author Michal 'justptT' Nowierski
- * @author Robert 'rochan' Chan
- * @ownership Content X-Wing
- */
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+
+import java.util.Map;
+
 public class HubsTests extends NewTestTemplate {
```

Comment:
If it's not ours then why we have X-Wing annotation above test methods.
And if hubs are not ours then why this one is ours: https://github.com/Wikia/selenium-tests/pull/896/files#diff-44577948b72e18df23b52e6290d6e66dR15

Do we owe hubs edition but not hubs themselves?